docs/ref/checks.txt

```diff
@@ -84,6 +84,14 @@ Django's system checks are organized using the following tags:
 * ``templates``: Checks template related configuration.
 * ``caches``: Checks cache related configuration.
 * ``urls``: Checks URL configuration.
+* ``database``: Checks any database-related connection issues. Database checks
+  are not run by default. They are only run by the :djadmin:`migrate` command
```

Comment:
We could add the rationale "are not run by default because they do more than static code analysis as regular checks do."